vg/draw/canvas.go

```diff
@@ -279,14 +317,16 @@ func (c *Canvas) Y(y float64) vg.Length {
 // Crop returns a new Canvas corresponding to the receiver
 // area with the given number of inches added to the minimum
 // and maximum x and y values of the Canvas's Rectangle.
-func (c Canvas) Crop(minx, miny, maxx, maxy vg.Length) Canvas {
+// Note that cropping the right and top sides of the canvas
+// requires specifying negative values of xmaxcrop and ymaxcrop.
+func (c Canvas) Crop(xmincrop, ymincrop, xmaxcrop, ymaxcrop vg.Length) Canvas {
```

Comment:
In this context, the word crop means to "make smaller" or "remove material". In this function, if I want to remove 2 cm from the right of the canvas, which at least I would think of as cropping +2 cm, I have to enter -2 cm as the argument for maxx, which is counterintuitive, at least to me. Although the documentation as is is technically correct, I had to originally read it multiple times before figuring out why I wasn't getting the expected results.